doc/src/manual/embedding.md

```diff
@@ -13,6 +13,7 @@ We start with a simple C program that initializes Julia and calls some Julia cod
 
 ```c
 #include <julia.h>
+JULIA_DEFINE_FAST_TLS() // only define this once, in an executable (not in a shared library) if you want fast code.
```

Comment:
It's up to us (well, and sometimes the platform ABI) where we decide to implement it.